docs/user-guide/user-roadmap-zowe-cli copy.md

```diff
@@ -0,0 +1,161 @@
+
+   The content also contains the web help for all Zowe ecosystem-conformant plug-ins that contributed to this website.
+   
+   You can read an interactive online version, download a PDF document, or download a ZIP file containing the HTML for the online version:
+   - <b><a href="/stable/web_help/index.html" target="_blank">Browse online</a></b>
+   - <b><a href="/stable/CLIReference_Zowe.pdf" target="_blank">Download CLI reference in PDF format</a></b>
+   - <b><a href="/stable/zowe_web_help.zip" target="_blank">Download CLI reference in ZIP format</a></b>
+
+- **Best practices**
+
+   Are you looking for recommendations and tips on how to best use Zowe CLI to meet your needs? These resources provide best practices recommendations.
+   
+   - [**Blog: Zowe CLI Tips & Tricks**](https://medium.com/modern-mainframe/zowe-cli-tips-tricks-79607b8dbd4e)
+
+- [**Zowe CLI and TSO commands**](https://medium.com/zowe/zowe-ci-and-tso-commands-14e5445fca1e)
+
+   This blog shows how to configure and use the TSO command feature of Zowe.
+
+## Developing a Zowe CLI plug-in
+
+> Zowe skill level: Advanced
+
+Zowe CLI extenders can build plug-ins that provide new commands.
+
+- [**Developing for the CLI**](../extend/extend-cli/cli-developing-a-plugin.md) 
+
+   Learn about developing for Zowe CLI.
+
+- [**Zowe CLI core repository**](https://github.com/zowe/zowe-cli)
+
+   If you want to start working with the code immediately, check out this code repository. 
+
+- [**Tutorials**](../extend/extend-cli/cli-devTutorials.md#tutorials)
+
+   Follow these tutorials to get started working with a sample plug-in.
+
+## Contributing to Zowe CLI
+
+> Zowe skill level: Advanced
+
+- [**Contributing guidelines**](https://github.com/zowe/zowe-cli/blob/master/CONTRIBUTING.md)
+
+   This document is a summary of conventions and best practices for development within Zowe CLI or development of Zowe CLI plug-ins. The guidelines contain critical information about working with the code, running, writing, and maintaining automated tests, developing consistent syntax in your plug-in, and ensuring that your plug-in integrates with Zowe CLI properly.
+
+- [**Conformance Program**](../extend/zowe-conformance-program.md)
+   
+  This topic introduces the Zowe Conformance Program. Conformance provides Independent Software Vendors (ISVs), System Integrators (SIs), and end users greater confidence that their software will behave as expected. As vendors, you are invited to submit conformance testing results for review and approval by the Open Mainframe Project. If your company provides software based on Zowe CLI, you are encouraged to get certified today.
+
+- [**Blog: Zowe Conformance Program Explained**](https://medium.com/zowe/zowe-conformance-program-7f1574ade8ea)
+
+   This blog describes the Conformance Program in more details.
+
+## Troubleshooting and support
+
```

Comment:
Skill level: Advanced?

Reply:
@zFernand0 -- we're no longer using these skill levels in the non-copy version of this file